src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs

```diff
@@ -3015,7 +3015,7 @@ internal TypeDescriptionNode(TypeDescriptionProvider provider)
                 IServiceProvider? provider,
                 [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)] Type objectType,
                 Type[]? argTypes,
-                object[]? args)
+                object?[]? args)
```

Comment:
The corresponding ref file should also be changed in `src\libraries\System.ComponentModel.TypeConverter\ref`.

Reply:
Thanks.